packages/maskbook/src/resources/NearIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core'
+
+export function NearIcon(props: SvgIconProps) {
```

Comment:
Hi I suggest to add new icons at `packages\icons`

Reply:
plz use https://aka.pw/svgomg reduce size

Reply:
Thanks @septs and @Jack-Works